src/server/game/Handlers/CharacterHandler.cpp

```diff
@@ -1849,11 +1850,21 @@ void WorldSession::HandleEquipmentSetUse(WorldPacket& recvData)
                 msg = _player->CanStoreItem(NULL_BAG, NULL_SLOT, sDest, uItem, false);
                 if (msg == EQUIP_ERR_OK)
                 {
+                    savedItems.emplace_back(std::make_unique<SavedItem>(uItem, dstpos));
                     _player->RemoveItem(INVENTORY_SLOT_BAG_0, i, true);
                     _player->StoreItem(sDest, uItem, true);
                 }
                 else
+                {
                     _player->SendEquipError(msg, uItem, nullptr);
+                    _player->SendEquipError(EQUIP_ERR_ITEMS_CANT_BE_SWAPPED,uItem,nullptr);
```

Comment:
Maybe could replace these with setting a variable to `4` and using that variable on L1894 instead of the `0`? Would probably want to declare it probably more at L1832 or something with a value of `0`, at least that'd be my approach. Removes the need for these equip errors and seems to be the Blizzlike message.

Reply:
Oh, sorry, I was thinking that there were 2 different errors. I was looking in the enums if there was a message like you said in the issue, but I didn't find it. My bad. It's my first time contributing here. I will try your approach. Thank you 😊.